cli/flox/doc/flox-activate.md

```diff
@@ -10,99 +10,103 @@ flox-activate - activate environments
 
 # SYNOPSIS
 
-flox [ `<general-options>` ] activate [ `<options>` ] [ -- `<command>` [ `<argument>` ] ]
+```
+flox [ <general-options> ] activate
+     [-d=<path> | -r=<owner/name>]
+     [ -t ]
```

Comment:
note/question (nb): we do have a (hidden flag) `--in-place` to force printing the script.

Do we want to make this official @ghudgins @jennymahmoudi ?

I'm not strongly attached to the name (@mkenigs preferred `--print-script`), nor that it is public

Reply:
can you put `for debugging` somewhere in the description and put it towards the end? I think @mkenigs' name is a bit more descriptive. no strong preference if you man page it or not. If a customer could find it useful to debug what flox is doing, doc it

Reply:
Done